src/components/organisms/Navbar/Navbar.js

```diff
@@ -147,12 +147,10 @@ export default function Navbar() {
                         <ul className="items-center justify-center space-y-8 lg:flex lg:space-x-5 xl:space-x-9 lg:space-y-0">
                             <li className="text-white">
                                 <div>
-                                    <Link className="peer py-2 text-white" href="/OurTechnology">{ourTecnology?.section}</Link>
```

Comment:
since `/OurTechnology` route is being replaced with `/ai`, I think it's fair to consider looking at whether the`/OurTechnology` page will be used anywhere else. Can you double check with designers?

If not, can you remove the entire page including its dependencies like images, types definitions, etc? Make sure those dependencies are not used elsewhere though.

Can you also confirm with them that by clicking on "Technology" on the navbar and footer, it goes to `/ai`?

Also, I noticed that the footer nav goes to `/OurTechnology` and not `/ai`.
Lmk if you have any questions.

Reply:
will do